quarkus-workshop-super-heroes/super-heroes/ui-super-heroes/src/main/java/io/quarkus/workshop/superheroes/ui/Config.java

```diff
@@ -1,4 +1,12 @@
 package io.quarkus.workshop.superheroes.ui;
 
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+/*
+Why do we need to register this for reflection? Normally this would be automatic if
+we return it from a REST endpoint, but because we're handling our own
+object mapping, we need to do our own registering.
+ */
+@RegisterForReflection
 public record Config(String API_BASE_URL, boolean CALCULATE_API_BASE_URL) {
```

Comment:
Why are you using upper cases here?

Reply:
Still trying to decide if the amount of code needed to fit it is worth it, since this case isn't one of the Jackson built-in ones. However, I have discovered that the name of this case is screaming snake case, and feel the need to share that delightful knowledge.

Reply:
Oh wait, looks like it made it in in Jackson 2.13.0: https://github.com/FasterXML/jackson-databind/issues/3238